docs/cli-manual.md

```diff
@@ -0,0 +1,252 @@
+# The TiDB Kubernete Contorl(tkctl) User Manual
+
+> **Disclaimer**: The tkctl CLI tool is currently **Alpha**. The design and sub-commands may change in the future, use at your own risk.
+
+The TiDB Kubernetes Control(tkctl) is a command line utility for TiDB operators to operate and diagnose their TiDB clusters in kubernetes.
+
+- [Installation](#installation)
+    - [Bash Completion](#bash-completion)
+    - [Kubernetes Configuration](#kubernetes-configuration)
+- [Commands](#commands)
+    - [tkctl version](#tkctl-version)
+    - [tkctl list](#tkctl-list)
+    - [tkctl use](#tkctl-use)
+    - [tkctl info](#tkctl-info)
+    - [tkctl get](#tkctl-get)
+    - [tkctl debug](#tkctl-debug)
+    - [tkctl ctop](#tkctl-ctop)
+    - [tkctl help](#tkctl-help)
+    - [tkctl options](#tkctl-options)
```

Comment:
[markdown-toc](https://github.com/jonschlinkert/markdown-toc) is a convenient tool to generate ToC for markdown docs:

```
markdown-toc --no-firsth1 docs/cli-manual.md
```